src/core/ProgressBar/BaseProgressBar/ProgressBar.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+import { ThemedBackground } from '../../../utils/storybook';
+import { ProgressBar } from './ProgressBar';
+
+storiesOf('ProgressBar/Base ProgressBar', module)
+  // Litmus Portal
+  .add('Litmus Portal', () => (
+    <ThemedBackground platform="litmus-portal" row>
```

Comment:
You would only require `row` here when you have more than one component wrapped in a ThemedBackground (only when you want those components to be aligned in a row alignment)